documentation/azure-powershell-modules.md

```diff
@@ -42,6 +42,7 @@ Custom Resource Providers             | `Az.CustomProviders`
 Data Box                              | `Az.DataBox`                            | [![DataBox]][DataBoxGallery]
 Data Box Edge (Stack Edge)            | `Az.DataBoxEdge`                        | [![DataBoxEdge]][DataBoxEdgeGallery]
 Databricks                            | `Az.Databricks`                         | [![Databricks]][DatabricksGallery]
+DataDog                               | `Az.DataDog`                            | [![DataDog]][DataDogGallery]
```

Comment:
There should be another line of code after line 170 to define "DataDogGallery".
This doesn't affect release, so it can be fixed later